docs/tutorials/add-catalog.md

```diff
@@ -57,7 +57,7 @@ This catalog is distributed as an image [quay.io/operatorhubio/catalog](https://
         type: Image
         image:
           ref: quay.io/operatorhubio/catalog:latest
-          pollInterval: 10m
+          PollIntervalMinutes: 10
```

Comment:
```suggestion
          PollIntervalMinutes: 10
```

Here and in other places "P" should not be capitalised.

Let's just revert this stuff and address it separately in https://github.com/operator-framework/operator-controller/pull/1443. Docs site is not versioned: we deploy from `main`.

Reply:
Good CATCHER !!!